newIDE/app/src/UI/CorsAwareImage.js

```diff
@@ -38,7 +38,6 @@ const addSearchParameterToUrl = (
 export const CorsAwareImage = (props: Props) => (
   <img // eslint-disable-line jsx-a11y/alt-text
     {...props}
-    crossOrigin="anonymous"
```

Comment:
I seem to remember you explained to me why this had to be changed but I'm not sure I remember why.

Reply:
Anonymous was not sending the cookies when fetching images, which we need now (when it's an image hosted on a gdevelop.io bucket, the cookie will be sent)